### PR TITLE
fix(plugin-meetings): browser-tests-local

### PR DESCRIPTION
**Pull request by Shreyas281299**
# COMPLETES #< INSERT LINK TO ISSUE >

## This pull request addresses

Browser tests were failing when run locally.

## by making the following changes
Browser tests are now running and passing. 2 tests are flaky which fails sometimes.
users "alice", "bob", and "chris" add media - this test suite fails but runs perfectly in pipeline

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ skipInNode(describe)('plugin-meetings', () => {
         })
     );
 
-    it('Alice starts a space meeting', () =>
+    it('Alice starts a space meeting', async () =>
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js]: Why async when I do not see an await inside?

File: packages/@webex/plugin-meetings/test/integration/spec/journey.js [hunk 2]
```diff
@@ -239,7 +239,7 @@ skipInNode(describe)('plugin-meetings', () => {
 
     // Enabled when config.enableUnifiedMeetings = true
     xdescribe('Conversation URL', () => {
-      describe('Successful 1:1 meeting', () => {
+      describe('Successful 1:1 meeting', async () => {
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/test/integration/spec/journey.js]: Why async in describe? Where do we await for this?

Comment (Shreyas281299) [reply packages/@webex/plugin-meetings/test/integration/spec/journey.js]: I missed this. This is not required. Removed

File: packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js (hunk 1 above)
Comment (Shreyas281299) [reply packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js]: Same this is also not required. removed this. This is flaky thats why I had put the async

**Review by arun3528 — APPROVED** [review]

**Pull request merged**